### ISSUE TITLE
Perform routing over "white areas".

### ISSUE THREAD

**Issue by CoSoCo** [osmandapp/OsmAnd#20414]
### Describe the idea (required)

When there are loaded offline maps from the start area and the destination area, OsmAnd routing should not fail, but at least show a route to and from the constraints of the existing maps and show an estimated air line in the non-covered area.

### Tell us about the expected behaviour (required)

At least show a route to and from the constraints of the existing maps and show an estimated air line in the non-covered area.

### Tell us about alternatives you've considered (required)

In most areas, e.g. Germany, there is a outline survey map, showing the "big roads", so why not use them for routing, if no detailed map for this area is loaded?

### Context (optional)

"Calculate online" would not be an option for me, as my device can't go "online" / does not have an active SIM card.
And if I had the opportunity beforehand, a previous "download" would usually not work either, as my device doesn't have that much free memory.
That's why I suggest using the existing overview map for routing, at least roughly, or at least simply using the straight line in the "white area".
Example:
I want to go from a certain street in Cologne/Germany to a certain street in Berlin/Germany, but for memory reasons I only have the maps of Cologne and Berlin, but not the areas in between.
In Cologne, I am only interested in how to get from the starting point to the motorway in an east-north-east direction or, in other words, "How do I get out of this unknown city in the rough direction to Berlin". On the way in the right direction, I would then come across traffic signs that would guide me to the motorway to Berlin. I don't need OsmAnd for the motorway route itself, as there are signs there.
And just before or from the border of Berlin, OsmAnd should guide me the last stretch to my destination street.

Translated with DeepL.com (free version)

**Comment by yuriiurshuliak** [comment]
Related to https://github.com/osmandapp/OsmAnd/issues/20389

**Comment by CoSoCo** [comment]
> Related to #20389

I don't think so, because a diversion / indirection can't be found here, as there is no connecting map between the start and destination area.
But it would help in case of #20413 .
